packages/components/src/stories/Drawer.stories.tsx

```diff
@@ -0,0 +1,193 @@
+import React, { useState } from 'react';
+import type { Meta, StoryObj } from '@storybook/react';
+import Drawer from '../Drawer/Drawer';
+
+const meta = {
+  title: 'Drawer',
+  component: Drawer,
+  parameters: {
+    layout: 'centered',
+  },
+} satisfies Meta<typeof Drawer>;
+
+export default meta;
+
+type Story = StoryObj<typeof meta>;
+
+export const Top: Story = {
+  decorators: [
+    (Story) => {
+      const [isOpen, setIsOpen] = useState(false);
+
+      const handleDrawerOpen = () => {
+        setIsOpen(true);
+      };
+
+      const handleDrawerClose = () => {
+        setIsOpen(false);
+      };
+
+      return (
+        <>
+          <button onClick={handleDrawerOpen}>top</button>
+          <Story
+            isOpen={isOpen}
+            onClose={handleDrawerClose}
+            args={{
+              anchor: 'TOP',
+              isOpen: isOpen,
+              onClose: handleDrawerClose,
+              children: (
+                <>
+                  <Drawer.Panel style={{ color: 'white' }}>
+                    첫 번째 메뉴
+                  </Drawer.Panel>
+                  <Drawer.Panel>두 번째 메뉴</Drawer.Panel>
+                  <Drawer.Panel>세 번째 메뉴</Drawer.Panel>
+                </>
+              ),
+            }}
+          />
+        </>
+      );
+    },
+  ],
```

Comment:
decorator 대신 render를 활용하면 스토리북 컨트롤 패널을 통해 자동적으로 anchor를 선택하게 할 수 있어요 👍 

```suggestion
  render: ({ anchor }) => {
    const [isOpen, setIsOpen] = useState(false);

    const handleDrawerOpen = () => {
      setIsOpen(true);
    };

    const handleDrawerClose = () => {
      setIsOpen(false);
    };

    return (
      <>
        <button onClick={handleDrawerOpen}>{anchor}</button>
        <Drawer
          isOpen={isOpen}
          onClose={handleDrawerClose}
          anchor={anchor}
        >
          <Drawer.Panel style={{ color: 'white' }}>
            첫 번째 메뉴
          </Drawer.Panel>
          <Drawer.Panel>두 번째 메뉴</Drawer.Panel>
          <Drawer.Panel>세 번째 메뉴</Drawer.Panel>
        </Drawer>
      </>
    );
  },
```